nuxt-frontend/components/backend-development/PartnerWithUsSection.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <section class="bg-[#121212] 3xl:outer-container">
+    <div class="py-14 lg:py-[7.5rem] container text-white">
+      <!-- ---------------------MOBILE UI START------------------ -->
+      <div class="flex flex-col gap-8 lg:hidden">
+        <h2 class="header-1 text-center">
+          Why should you partner with Canopas?
+        </h2>
+        <div class="flex flex-col gap-6" v-for="item in items" :key="item">
+          <div class="flex gap-4">
+            <img
+              :src="item.image"
+              class="w-9 h-9 object-contain"
+              :alt="item.title"
+            />
+            <div class="flex flex-col gap-3">
+              <span class="sub-h1-semibold">{{ item.title }}</span>
+              <span
+                class="sub-h3-regular text-white/[0.8]"
+                v-html="item.description"
+              ></span>
+            </div>
+          </div>
+        </div>
+        <nuxt-link
+          class="gradient-btn primary-btn mt-0"
+          to="/contact"
+          @click.native="$mixpanel.track('tap_backend_partner_with_us_cta')"
+        >
+          <span class="sub-h3-semibold">Let’s Work Together</span></nuxt-link
+        >
+      </div>
+      <!-- ---------------------MOBILE UI END-------------------- -->
+
+      <!-- ---------------------DESKTOP UI START------------------ -->
+      <div class="hidden lg:block stickyparent h-[150vh]">
+        <div class="sticky top-0 h-[110vh] xl:top-4 xl:h-[100vh] main">
+          <div
+            class="text-white/[0.12] absolute -translate-y-1/2 top-1/2"
+            :class="{
+              '!ease-in-out !flex flex-1': isAnimated,
+            }"
+          >
+            <div
+              class="flex-col my-auto opacity-100 transition-all delay-300 transform ease-in-out"
```

Comment:
i don't think that text should animate in this way
![canopas-website](https://github.com/canopas/canopas-website/assets/123158106/af0d26e9-c3f8-47d6-a11c-4080a3563c91)